pkg/validation/app_test.go

```diff
@@ -28,7 +29,7 @@ func Test_ValidateApp(t *testing.T) {
 		expectedErr string
 	}{
 		{
-			name: "case 0: flawless flow",
+			name: "flawless flow",
```

Comment:
Changing this is fine with me. I find the numbers useful when scanning the code but it makes them a bit harder to maintain.

Reply:
Agree, it's easier to maintain without the numbers, but I'm still not sure if that justifies the change :D I can always easily revert it back if we agree we like it better with the numbers.

Reply:
I like removing the numbers here - it makes adding tests easier IMO

Reply:
Ok then, so I guess we all agree to leave it?

Reply:
Fine with me :)